XenAdmin/ConsoleView/VNCTabView.cs

```diff
@@ -768,7 +779,7 @@ private string GuessNativeConsoleLabel(VM source)
                     if (osString != null)
                     {
                         if (osString.Contains("Microsoft"))
-                            label = UseRDP;
+                            label = (Helpers.CreedenceOrGreater(targetHost) && !RDPEnabled) ? enableRDP : UseRDP;

```

Comment:
I still cannot find CreamOrGreater in master branch. Do I still need to wait for some time?